app/src/pages/inside/projectSettingsPageContainer/content/integrations/integrationsList/integrationInfo/integrationInfo.jsx

```diff
@@ -313,7 +314,7 @@ export const IntegrationInfo = (props) => {
             <IntegrationSettingsComponent
               data={updatedData}
               onUpdate={onUpdate}
-              goToPreviousPage={goBackHandler}
+              goToInitialPage={goToInitialPage}
```

Comment:
Also leave naming here and inside of all components as is, just provide necessary function here.